packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -331,19 +334,25 @@ const Docs: () => JSX.Element = () => {
       </div>
 
       <div ref={refDisplayAttachments}>
-        <Heading>Display Attachments with Messages</Heading>
+        <Heading>Display Messages with Inline Attachments</Heading>
```

Comment:
Should it be just `Display Messages with Attachments`?

Reply:
sure